.github/workflows/docker-backend.yml

```
@@ -6,22 +6,16 @@ on:
         type: string
         default: docker.nexus.famedly.de
 
-      # Name of service. Empty for root service, needs to be unique for services from subfolders. 
+      # Name of service.
       # Will be used as a last part of container path and matched with version tags
```

if path is specified, right?

---

I don't understand?

---

the comment should probably be extended:
```suggestion
      # Will be used as a last part of container path and matched with version tags if path is specified
```